python/sparkdl/estimators/keras_image_file_estimator.py

```diff
@@ -0,0 +1,301 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import base64
+import hashlib
+
+import numpy as np
+import keras    
+
+from pyspark.ml import Estimator, Transformer
+from pyspark.ml.param import Param, Params, TypeConverters
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import udf
+
+from sparkdl.image.imageIO import imageArrayToStruct, imageStructToArray, imageSchema
+from sparkdl.transformers.param import (
+    keyword_only, HasInputCol, HasOutputCol, SparkDLTypeConverters)
+from sparkdl.transformers.tf_image import TFImageTransformer, OUTPUT_MODES
+import sparkdl.utils.jvmapi as JVMAPI
+import sparkdl.utils.keras_model as keras_utils
+
+if False:
```

Comment:
Some version of this will go into test and docs. 
Just put them here to show how it is being developed.